### PR TITLE
fixes #2697 - HostMailer summary is broken

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

**Comment by GregSutcliffe** [comment]
I hit this the other day and forgot to send the exact same oneliner. Good catch @isratrade ! :+1:

**Comment by skottler** [comment]
Merged as 7ff08812e42c. @domcleal can you cherry-pick to 1.2-stable?

File: lib/tasks/reports.rake [hunk 1]
```diff
@@ -77,6 +77,6 @@ namespace :reports do
 
     options[:email] = ENV['email'] if ENV['email']
 
-    HostMailer.deliver_summary(options)
+    HostMailer.summary(options)
```
Comment (domcleal) [inline lib/tasks/reports.rake:80]: This should have `.deliver` on the end, else the message doesn't get sent.

**Comment by domcleal** [comment]
There's another bug in the summary introduced by feacea35 where the subject's wrong, see app/models/host_mailer.rb:65, it says 'subnet' instead of 'subject'.  Perhaps you can fix that too?

As this is already on develop, it might be best to send a followup PR referencing this issue number for the two fixes.  Cheers!

**Comment by isratrade** [comment]
previously merged and new PR https://github.com/theforeman/foreman/pull/738 submitted

**Pull request closed without merging**